src/node_crypto.h

```diff
@@ -544,10 +552,15 @@ class Hash : public BaseObject {
       : BaseObject(env, wrap),
         initialised_(false) {
     MakeWeak<Hash>(this);
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+    mdctx_ = reinterpret_cast<EVP_MD_CTX *>(malloc(sizeof(EVP_MD_CTX)));
```

Comment:
This will break 1.0.2. It needs to call `EVP_MD_CTX_init(ctx_)` to zero-initialize everything. But you're better off calling `EVP_MD_CTX_create` here and changing the above to `EVP_MD_CTX_destroy`.